### DISCUSSION TITLE
Add many members to Organisation/Team

### DISCUSSION THREAD

**Discussion by Churros396** [opengisch/QField#5046] [Q&A]
Is there a method to add multiple members via a table or something similar on QField Cloud ? Same for the Teams section. 

Thank you for your insights.

**Comment by SeqLaz** [comment]
Hello @Churros396, I'm sorry to tell you that this functionality is not possible yet.

**Reply by Churros396** [reply]
Do you know if it is on the road map as a feature? Thank you for your fast reply.

**Reply by SeqLaz** [reply]
We do not have plans to integrate this feature at the moment, but we are taking it into consideration. We apologize for the inconvenience.

**Reply by Churros396** [reply]
Thx, it would be a great quality of life feature.

**Reply by suricactus** [reply]
@Churros396 feel free to add your suggestion here https://qfield.canny.io/qfieldcloud-feature-requests
